### PR TITLE
Regen Ledger v2.0 Upgrade - Signalling Proposal

### PR DESCRIPTION
**Pull request by ryanchristo**
This pull request adds the signalling proposal for the Regen Ledger v2.0 upgrade.

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Looks great

**Review by dpdanpittman — APPROVED** [review]
Thanks for creating a pdf, looks good to me.

File: proposals/2021-09-regen-ledger-v2-signalling/credit-class-creator-process.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+### Credit Class Creator
+
+A credit class creator is an address with the authority to create credit classes. The list of approved credit class creators is stored as an on-chain parameter that can only be updated through the governance process. Credit classes can be created by executing a transaction on-chain with the required credit type, metadata, and list of approved issuers. Upon creation of a credit class, the credit class creator is set as the admin for the given credit class.
+Credit Class Administrator
+The credit class administrator (`CreditClassAdmin` as represented in code) is defined within a credit class as the address with the authority to manage and update the credit class. The credit class admin will have the ability to transfer the admin role to another address, manage the list of credit class issuers, and change credit class metadata.
```
Comment (ryanchristo) [inline proposals/2021-09-regen-ledger-v2-signalling/credit-class-creator-process.md]:
```suggestion

### Credit Class Administrator

The credit class administrator (`CreditClassAdmin` as represented in code) is defined within a credit class as the address with the authority to manage and update the credit class. The credit class admin will have the ability to transfer the admin role to another address, manage the list of credit class issuers, and change credit class metadata.
```

File: proposals/2021-09-regen-ledger-v2-signalling/proposal.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+{
+    "title": "Regen Ledger v2.0 Upgrade - Signalling Proposal",
```
Comment (ryanchristo) [inline proposals/2021-09-regen-ledger-v2-signalling/proposal.json:3]:
```suggestion
    "type": "Text",
    "title": "Regen Ledger v2.0 Upgrade - Signalling Proposal",
```

File: proposals/2021-09-regen-ledger-v2-signalling/proposal.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (ryanchristo) [inline proposals/2021-09-regen-ledger-v2-signalling/proposal.json]:
```suggestion
```

File: proposals/2021-09-regen-ledger-v2-signalling/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+### Authz Module
+
+The authz module enables a granter to grant an authorization to a grantee that allows the grantee to execute a message on behalf of the granter. For more information about the authz module, see the specification:
+
+- [Authz Module Specification](https://docs.cosmos.network/master/modules/authz/)
+
+### Feegrant Module
+
+The feegrant module enables the ability for a granter to grant an allowance to a grantee where the allowance is used to cover fees for sending transactions. For more information about the feegrant module, see the specification:
+
```
Comment (clevinson) [inline proposals/2021-09-regen-ledger-v2-signalling/README.md:29]:
```suggestion

### Permanent Locked Accounts

Regen Ledger v2.0 supports the on-chain creation of permanent locked accounts through the `MsgCreatePermanetLockedAccount` Msg and via CLI. These special types of accounts are intended to be used by Regen Foundation for seeding Community Staking DAOs, wherein the initial REGEN funds distributed to these accounts must be permanently locked and only usable for governance and staking.

For more information see [regen-ledger#188](https://github.com/regen-network/regen-ledger/issues/188)
```

How is this?

File: proposals/2021-09-regen-ledger-v2-signalling/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+- [Feegrant Module Specification](https://docs.cosmos.network/master/modules/feegrant/)
+
```
Comment (clevinson) [inline proposals/2021-09-regen-ledger-v2-signalling/README.md:31]:
```suggestion

### Permanent Locked Accounts

Regen Ledger v2.0 supports the on-chain creation of permanent locked accounts through the `MsgCreatePermanetLockedAccount` Msg and via CLI. These special types of accounts are intended to be used by Regen Foundation for seeding Community Staking DAOs, wherein the initial REGEN funds distributed to these accounts must be permanently locked and only usable for governance and staking.

For more information see [regen-ledger#188](https://github.com/regen-network/regen-ledger/issues/188)

```

@ryanchristo  How is this?

**Review by dpdanpittman — APPROVED** [review]
Looks good

**Pull request merged**